dvc/command/experiments.py

```diff
@@ -1249,6 +1250,12 @@ def add_parser(subparsers, parent_parser):
         action="store_true",
         help="Remove all committed experiments.",
     )
+    remove_group.add_argument(
+        "-r",
+        "--git-remote",
+        metavar="<git_remote>",
```

Comment:
Should be something non-repetitive. AFAIR we try to use pseudo types here? In this case `<text>` or maybe `<name>`

Reply:
`<remove name or url>` is this too long?

Reply:
We are using `metavar="<git_remote>"` already for other `exp push/pull`. If you think it's unhelpful @jorgeorpinel, can we extract to a separate issue and keep as is for now for consistency and to not block this pr?

Reply:
Yeah it's a mix in the codebase. At some point I reviewed a lot of metavars and changed their descriptions to metatype annotations but maybe missed some and new ones may be repetitive/unhelpful here. Maybe @skshetry wants to consider this for https://github.com/iterative/dvc/issues/5392 🙂 Up to you guys